packages/UI/src/components/organisms/PositionCard/index.ts

```diff
@@ -0,0 +1,10 @@
+export { default } from './PositionCard';
+
+export interface PositionCardProps {
+  // interaction: boolean;
+  // status: boolean;
+  interaction: 'all-h' | 'all-v' | 'save-only' | 'none';
+  style: 'default' | 'minimal' | 'mobile' | 'long';
+  buttonStyle: 'a' | 'b';
+  status: 'default' | 'open' | 'closed';
```

Comment:
I think this is more a boolean right? @Nouri-Anouar

Reply:
Yes I agree, a boolean is easier for conditional logic.